src/main/java/net/atos/zac/app/klanten/KlantenRESTService.java

```diff
@@ -199,6 +199,16 @@ public List<RESTRoltype> listBetrokkeneRoltypen(@PathParam("zaaktypeUuid") final
         );
     }
 
+    @GET
+    @Path("roltype")
+    public List<RESTRoltype> listRoltypen() {
```

Comment:
We introduce a new endpoint here? Why exactly? I thought this PR was just about adding betrokkenen in our test data? I'm a bit surprised to see any code changes.